File: Beginner Programmer - 1/Second Highest Digit/solution.js
'use strict';

module.exports = {
  secondHighestDigit
};

function secondHighestDigit(input) {
    // Your code goes here
    let digits = input.split("").filter(value => !isNaN(value))
    if (digits.length < 2){
        return -1
    }
    return digits.sort().map(value=> parseInt(value)).reverse()[1];
    
}

File: Beginner Programmer - 1/Second Highest Digit/solution.py
class Solution:

    def second_highest_digit(self, input_string):
        # Your code goes here
        digits = sorted(filter(lambda x: x.isdigit(), input_string), reverse=True)
        if len(digits) < 2:
            second = -1
        else:
            second = int(digits[1])
        return second